src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1442,13 +1442,27 @@ def _configure_security_policy_storage_params(arg_ctx):
                 'deleted_time',
                 'target_managed_database_name',
                 'target_managed_instance_name',
-                'restore_point_in_time'
+                'restore_point_in_time',
+                'long_term_retention_backup_resource_id',
+                'restore_mode'
             ])
 
-        c.argument('deleted_time',
-                   options_list=['--deleted-time'],
-                   help='If specified, restore from a deleted database instead of from an existing database.'
-                   ' Must match the deleted time of a deleted database on the source Managed Instance.')
+        c.argument(
+            'restore_mode',
+            options_list=['--restore-mode', '--mode'],
+            required=True,
```

Comment:
Adding a new required param is a breaking change :( we shouldn't do this.

Reply:
reverted this